install/resources/mybb_theme.xml

```diff
@@ -4373,7 +4373,7 @@ $(".forumjump").change(function() {
 <thead>
```

Comment:
Template versioning should be updated when editing templates. The next scheduled version is 1.8.16, so it should be updated to `1816`. I can't add a comment to the exact line which is 4372 (`index_boardstats`).

Reply:
@Shade- I don't have push access to the PR as I have deleted the main fork along with this branch.
If you see its okay to merge I will create another back to back PR with correct revision numbers?
Is that okay? Else I have to create the complete PR fresh ...

Reply:
As long as you reference this PR with the new one, it's ok.